templates/angular.webapp/src/app/tech/tech.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import {TechService} from './tech.service';
+import {Technology} from './tech.model';
+
+@Component({
+  selector: 'app-tech',
+  templateUrl: './tech.component.html'
+})
+export class TechComponent implements OnInit {
+
+  technologies: Technology[] = [];
+
+  constructor(private readonly techService: TechService) { }
+
+  ngOnInit() {
+    this.techService.getTechnologies().subscribe(value => {
```

Comment:
I read that if we subscribe we need realize to unsubscribe. What do you think?

Reply:
This is generally true but is not needed in every case, because the Observable of a http request is completed after the request is done, this is handled by the angular httpClient, so there is no need to unsubscribe explicitly in this case

Reply:
Thank you. It is necessary and useful information